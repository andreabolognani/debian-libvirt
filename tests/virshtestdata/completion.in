complete -- ech
complete -- echo
complete -- echo ''
complete -- echo --xm --s
complete -- echo --nonexistant-arg --xml
complete -- echo --nonexistant-arg --xml ''
complete -- echo --nonexistant-arg --prefix
complete -- echo --nonexistant-arg --prefix ''
complete -- domstats --domain
complete -- domstats --domain ''
complete -- domstats --domain f
complete -- domstats --domain fc --domain fv
complete -- domstats --domain fv --domain fc ''
complete -- domstats --domain fc fv
complete -- domstate ''
complete -- domstate fv
