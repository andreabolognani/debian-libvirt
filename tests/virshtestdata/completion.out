echo

echo

--shell
--xml
--split
--err
--prefix
--string

--shell
--split
--string

--xml

--shell
--xml
--split
--err
--prefix
--string

--prefix

--shell
--xml
--split
--err
--prefix
--string

--domain

test



--state
--cpu-total
--balloon
--vcpu
--interface
--block
--perf
--iothread
--memory
--dirtyrate
--vm
--list-active
--list-inactive
--list-persistent
--list-transient
--list-running
--list-paused
--list-shutoff
--list-other
--raw
--enforce
--backing
--nowait
--domain


--domain
--reason



## Exit code: 1
